core/src/main/java/com/google/cloud/sql/core/CoreSocketFactory.java

```diff
@@ -152,24 +140,6 @@ public static synchronized CoreSocketFactory getInstance() throws IOException {
     return coreSocketFactory;
   }
 
-  @VisibleForTesting
```

Comment:
Let's move all this re-formatting into a separate PR -- it makes it hard to see what's actually changing here.